test/compat-shallow-renderer/PreactShallowRendererMemo_test.tsx

```diff
@@ -0,0 +1,1606 @@
+/**
```

Comment:
Haha yea, agreed. I was surprised as well. It looks like many of the tests from `PreactShallowRenderer_test.tsx` are duplicated here but using memo. I suspect this is because the react shallow renderer has to implement the memo behavior itself ([react-shallow-renderer runs comparer function](https://github.com/enzymejs/react-shallow-renderer/blob/master/src/ReactShallowRenderer.js#L575-L582)).